pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Pull Requests
+
+Please be sure you have read our documentation regarding contributing
+here: https://docs.pulpproject.org/index.html
+
+There you can read a Contribution Checklist and learn more about
+appropriate branches and rebasing among other things.
+
+Once you are ready to create a pull request:
+
+- [ ] If there is a bug for this issue, please title the pull request
+"<bug_id> - Short Message"
+- [ ] In the commit message or comment section for your pull request,
+please include a link to the issue.
+- [ ] If there is a Redmine issue associated with the commit, please
+add "closes #<issue number>" in your commit as this will set the
+issue to MODIFIED in the tracker upon merging.
+- [ ] Once you have created your pull request, please add a link to
+your pull request into the notes/comments for the relevant bug in the
+issue tracker.
```

Comment:
Also, the issue should be set to POST status.

Reply:
Good catch.  Happy to update this.  I was using the stable docs to write this, which is evidently out of date.